### DISCUSSION TITLE
How to use Adaptive Gradient Clipping in PL?

### DISCUSSION THREAD

**Discussion by sakvaua** [Lightning-AI/pytorch-lightning#6128] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Hello, PyTorch en-lightened gang! :)

How do I inject Adaptive Gradient Clipping (AGC from the https://arxiv.org/abs/2102.06171) into Pytorch Lightning trainer? Somewhere between the loss.backward and optimizer.step?
Thanks!
